index.bs

```diff
@@ -2580,6 +2586,14 @@ The [=remote end steps=] with |command parameters| are:
      * Otherwise, the details of how the browsing context is presented to the
        user are implementation defined.
 
+  1. If the value of the |command parameters|' <code>background</code> field is not present or false:
+
+    1. Let |activate result| be the result of [=activate a context=] with the newly created [=/browsing context=].
+
+    1. If |activate result| is an [=error=], return |activate result|.
+
+    Note: Do no invoke the [=/focusing steps=] for the created browsing context if background is true.
```

Comment:
I believe `background` needs to be tagged - not sure if it's `|background|` or `<code>background</code>`.